### PR TITLE
Deployment and notifications

### PR DESCRIPTION
**Pull request by Dmitry-GH**

### CODE REVIEW

File: push-notifications.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+# Useful articles
+
+You can start from [Best Practices for Push Notifications in React Native](https://www.wix.engineering/post/best-practices-for-push-notifications-in-react-native), and then check [repo](https://github.com/yogevbd/PushNotificationsTutorial) for the whole code from this article.
+
+Also, FireBase has a good article about setting up all iOS required stuff:
+[iOS Messaging Setup](https://rnfirebase.io/messaging/usage/ios-setup)
+
+After doing this right, you will have a working Push Notification, and it's enough for you in case if you will not use Firebase Cloud Messaging.
+
+Honestly, it will work with Firebase too, but you will not receive notifications in the Foreground app state (at least for the moment, when I'm writing this). To get this works, you can use this little hack:
```
Comment (tolgahan-arikan) [inline push-notifications.md]: "To get this works" -> "To get this to work"

File: deployment-pipeline.md [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+
+[Github](https://github.com/uptechteam/Sandbox-Mobile-App)
+
+[Github repo for Match](https://github.com/uptechteam/Sandbox-Mobile-App-ios-certs)
+
+You can also find it in `Google Play Console`, `Firebase`, `AppCenter`.
+
+Because of some problem with Notification setups (related to Apple Developer Keys) - I created the `iOS` version under my personal App Store Connect account. So, if you will need access to it - just send me PM (Dmitry Rudiuk).
+
+>* Match Repo pass: `Dimon4350489` (yeah, I know, the dumb password, but I can't change it now in an easy way, sorry)
```
Comment (tolgahan-arikan) [inline deployment-pipeline.md]: hahah, I think no need for `(yeah, I know, the dumb password, but I can't change it now in an easy way, sorry)` part, password is a password, nothing dumb with it :)

**Review by tolgahan-arikan — APPROVED** [review]
Looks great, thank you! <3

File: deployment-pipeline.md (hunk 2 above)
Comment (Dmitry-GH) [reply deployment-pipeline.md]: =D
Thanks for your review!

**Pull request merged**